src/Matcher.php

```diff
@@ -78,16 +105,6 @@ public static function compareMatches(Match $a, Match $b)
      */
     protected function getMatchers()
     {
-        // @todo change to dynamic
-        return [
-            Matchers\DateMatch::class,
-            Matchers\DictionaryMatch::class,
-            Matchers\ReverseDictionaryMatch::class,
-            Matchers\L33tMatch::class,
-            Matchers\RepeatMatch::class,
-            Matchers\SequenceMatch::class,
-            Matchers\SpatialMatch::class,
-            Matchers\YearMatch::class,
-        ];
+        return array_merge($this->defaultMatchers, $this->additionalMatchers);
```

Comment:
The `additionalMatchers` property contains string keys (the FQCN of the matcher) while the default matchers array has numeric indexes.

Should it be as follows to be consistent?

```suggestion
        return array_merge($this->defaultMatchers, array_values($this->additionalMatchers));
```